home.html

```diff
@@ -1,53 +1,91 @@
 <!DOCTYPE html>
 <html>
-	<head>
-		<meta charset="UTF-8">
-		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-		<meta http-equiv="X-UA-Compatible" content="ie=edge">
-		<link rel="stylesheet" href="css/style.css">
-		<title>Home</title>
-	</head>
-
-	<body>
-	
-		<div class = "wrapper">
-			
-			<!--<div class = "header">
-
-			</div>-->
-			
-			<div class = "nav">
-				<div class="hamburger-menu">
-					<input id="menu__toggle" type="checkbox" />
-					<label class="menu__btn" for="menu__toggle">
-					  <span></span>
-					</label>
-
-					<ul class="menu__box">
-					  <li><a class="menu__item" href="home.html">Home</a></li>
-					  <li><a class="menu__item" href="library.html">Library</a></li>
-					  <li><a class="menu__item" href="login.html">Login</a></li>
-					</ul>
-				</div>
+   <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <link rel="stylesheet" href="css/style.css">
+      <title>Home</title>
+   </head>
+   <body>
+      <div class = "wrapper">
+		  
+         <!--<div class = "header">
+         </div>-->
+		  
+         <div class = "nav">
+            <div class="hamburger-menu">
+               <input id="menu__toggle" type="checkbox" />
+               <label class="menu__btn" for="menu__toggle">
+               <span></span>
+               </label>
+               <ul class="menu__box">
+                  <li><a class="menu__item" href="home.html">Home</a></li>
+                  <li><a class="menu__item" href="library.html">Library</a></li>
+                  <li><a class="menu__item" href="login.html">Login</a></li>
+               </ul>
+            </div>
+			 
+            <center>
+               <p>navigation bar/burger nav</p>
+            </center>
+			 
+			 <p>logo</p>
+			 
+         </div>
+         <div class = "main_content">
+            
+			 <div class = "title1">
+               <br>
+				 <b>
+                  <center>
+					  <h1>Welcome</h1>
+                  </center>
+               </b>
+				 
+				<b>
+                  <center>
+                     <p>See new releases here</p>
+                  </center>
+               </b>
+				 
+            </div>
+			 
+            <!--Content Boxes-->
+			 
+            <div class = "content_2">
 				
-				<center><p>navigation bar/burger nav</p></center>
-
-			</div>
-			
-			<!--Content Boxes-->
-			<div class = "content_2">
-				<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
-				<center><p>welcome message</p></center><br><br>
+               <center>
```

Comment:
I have added image boxes that will showcase images of song albums in the database.